chapters/lexicalstructure.tex

```diff
@@ -247,6 +247,24 @@ \subsection{Strings}\label{strings}
 \end{lstlisting}
 \end{nonnormative}
 
+\subsection{Units of Literal Constants}\label{units-literal-constants}
+
+The following rules apply:
+- The result of `x + L`, `L + x`, `x - L`, `L - x`, `x*L`, `L*x`, `x/L`, where `x` is an expression with non-empty \lstinline!unit! attribute string `<s>` and `L` is an expression containing only literal constants, shall have the same \lstinline!unit! attribute string of `x`.
+- The result of `L/x` shall the the \lstinline!unit! attribute string `1/(<s>)`.
+- The result of `x^L` shall have the \lstinline!unit! attribute string of `product(fill(x, L))` if L is a positive Integer, or the unit of `1/product(fill(x, L))` if L is a negative Integer
```

Comment:
```suggestion
- The result of \lstinline!x^L! shall have the \lstinline!unit! attribute string of \lstinline!product(fill(x, L))! if L is a positive Integer, or the unit of \lstinline!1/product(fill(x, L))! if L is a negative Integer
```
The other option would be $x^L$ or something similar, but the current variant doesn't work.

Reply:
To discuss, if we use this variant we should use it more consistently.

Reply:
LGTM

Reply:
Looks like it's done already in the PR